iml-api/src/graphql.rs

```diff
@@ -813,6 +828,20 @@ impl MutationRoot {
 
         Ok(true)
     }
+
+    /// Delete a stratagem report
+    #[graphql(arguments(filename(description = "The report filename to delete")))]
+    async fn delete_stratagem_report(
+        _context: &Context,
+        filename: String,
+    ) -> juniper::FieldResult<bool> {
+        let mut path = get_report_path();
+        path.push(filename);
```

Comment:
What happens if filename is `../../*`?

Reply:
right, I'm currently thinking about that but wanted to push a first draft. Do you have any initial thoughts?

Reply:
Pushed an update.